.github/workflows/deploy-prod.yaml

```diff
@@ -105,13 +105,13 @@ jobs:
           method: 'GET'
 
       - name: Update Pull Request
-        uses: actions/github-script@v3
+        uses: actions/github-script@v6
         env:
           SUCCESS: '\u2705'
           FAILURE: '\u274C'
           INFRAMAP_URL: ${{ steps.inframap.outputs.imgurl }}
         with:
-          github-token: ${{ secrets.GITHUB_TOKEN }}
+          github-token: ${{ secrets.GH_TOKEN }}
```

Comment:
I believe that `GITHUB_TOKEN` is automatically set by Github so we probably want to use this rather than a manually set value